### DISCUSSION TITLE
Symbol type is not shown properly in TOC

### DISCUSSION THREAD

**Discussion by dodid** [mkdocstrings/python#147] [Q&A] [answered]
I tried to add the symbol type to TOC: https://dodid.github.io/minitrade/backtest/ But it's not displayed properly. How can I fix that? Thanks.

**Comment by pawamoy** [comment]
That's probably because you have markdown 3.6 installed. This version broke our symbol toc feature. We have a fix for it, but in the meantime you can try to downgrade it.

**Comment by dodid** [comment]
Appreciate your reply. I was on 3.5.2. I tried 3.5.0 and 3.4.0. However, both gave me the same result... Here is what I have:

```
% pip list | grep mk         
mkdocs                     1.5.3
mkdocs-autorefs            1.0.1
mkdocs-click               0.8.1
mkdocs-jupyter             0.22.0
mkdocs-material            8.5.11
mkdocs-material-extensions 1.3.1
mkdocstrings               0.20.0
mkdocstrings-python        1.9.0

% pip list | grep -i markdown
Markdown                   3.5
markdown-it-py             3.0.0
```

**Reply by pawamoy** [reply, marked as answer]
Ah sorry, yes, this one is fixed by upgrading mkdocs-material.

**Reply by dodid** [reply]
Fixed, thank you!
